src/toolkit/position.h

```diff
@@ -0,0 +1,187 @@
+#ifndef CYCLUS_SRC_TOOLKIT_POSITION_H_
+#define CYCLUS_SRC_TOOLKIT_POSITION_H_
+
+#include <string>
+
+#include "cyclus.h"
+
+namespace cyclus {
+namespace toolkit {
+/// @class Position
+/// The Position class is a basic class that stores the geographic location of
+/// each
+/// agent in latitude and longitude and follows the ISO
+/// 6709 standard. Allows string expression of Position objects as specified in
+/// ANNEX H of ISO 6709 found here: 'https://www.iso.org/standard/39242.html'
+/// or a quick simplified version found here
+/// 'https://en.wikipedia.org/wiki/ISO_6709'
+///
+/// Brief Description of ISO 6709:
+/// 1. Latitude comes before Longitude
+/// 2. North latitude is positive
+/// 3. East longitude is positive
+/// 4. Fraction of degrees (decimal values) is prefered in digital data
+/// exchange.
+///
+/// The class is adapted from 'https://github.com/jaime-olivares/coordinate'
+/// under the MIT License.
+///
+///   Copyright (c) 2015 Jaime Olivares
+///
+///   Permission is hereby granted, free of charge, to any person obtaining a
+///   copy of this software and associated documentation files (the "Software")
+///   , to deal in the Software without restriction, including without
+///   limitation the rights to use, copy, modify, merge, publish, distribute,
+///   sublicense, and/or sell copies of the Software, and to permit persons to
+///   whom the Software is furnished to do so, subject to the following
+///   conditions:
+///
+///   The above copyright notice and this permission notice shall be included
+///   in all copies or substantial portions of the Software.
+///
+///   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+///   OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+///   MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+///   NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+///   DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+///   OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+///   USE OR OTHER DEALINGS IN THE SOFTWARE.
+///
+/// Longitude and Latitude is stored as seconds of degrees. This was also
+/// adapted from 'https://github.com/jaime-olivares/coordinate'. This, according
+/// to the jaime-olivares, allows the coordinate elements such as degrees,
+/// minutes, and seconds to "remain on the integral portion of values, with the
+/// exception of decimal of seconds, avoiding loss of precision." This is
+/// calculated by multiplying decimal degrees by 3600.
+/// example: 05.2169 -> 18780.84
+
+class Position {
+  static const int DecimalSecondsMultiplier = 3600;
+
+ public:
+  enum StringFormat {
+    D = 1,
+    DM,
+    DMS,
```

Comment:
I don't think these enum value names are particularly readable?  Maybe it is because I am not a GIS expert but can you give them longer, more descriptive names please? Thanks!